spec/request/recipefoods_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Recipefoods', type: :request do
+  describe 'GET foods#index' do
+    before(:each) do
+      user = User.create name: 'Guy', email: 'hulk@example.com', password: '123456'
+      post user_session_path, params: { user: { email: user.email, password: user.password } }
+      get foods_path
+    end
+
+    it 'should return http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should render correct content' do
+      expect(response.body).to include('Recipe')
+    end
```

Comment:
- We also realised that so far everyone who has done the testing has  removed the confirmable and had an approval so we are not sure weather we should  remove it again since the  first code reviewer told us to add it back.  Please kindly assist.